.travis.yml

```diff
@@ -49,25 +49,25 @@ matrix:
           - mysql-server-5.5
           - mysql-client-core-5.5
           - mysql-client-5.5
-    - rvm: 2.0.0
+    - rvm: 2.4
       env: DB=mariadb55
       addons:
         mariadb: 5.5
         hosts:
           - mysql2gem.example.com
-    - rvm: 2.0.0
+    - rvm: 2.4
       env: DB=mariadb10.0
       addons:
         mariadb: 10.0
         hosts:
           - mysql2gem.example.com
-    - rvm: 2.0.0
+    - rvm: 2.4
       env: DB=mariadb10.1
       addons:
         mariadb: 10.1
         hosts:
           - mysql2gem.example.com
-    - rvm: 2.0.0
+    - rvm: 2.4
```

Comment:
I found out we should write "2.4.0"....

see: https://github.com/brianmario/mysql2/pull/821